spec/spec_helper.rb

```diff
@@ -16,10 +16,10 @@
 include SourceHelper
 include SharedData::Samples
 
-def create_answer metric: sample_metric, company: sample_company,
+def create_answer metric: sample_metric, company: sample_company, user: "Joe User",
                   content: "content", year: "2015", source: sample_source.name
   #content ||= "I'm fine, I'm just not happy."
```

Comment:
I think this emotional state is from Henry and probably outdated and can be deleted :wink: